docs/introduction/getting_started.rst

```diff
@@ -27,22 +48,37 @@ First we will construct two :ref:`site_documentation` objects for the :math:`^1\
     from mrsimulator.spin_system.tensors import SymmetricTensor
 
     # Create the Site objects
-    H_site = Site(isotope="1H")
+    H_site = Site(isotope = "1H")
     C_site = Site(
-        isotope="13C",
-        isotropic_chemical_shift=100.0,  # in ppm
-        shielding_symmetric=SymmetricTensor(
-            zeta=70.0,  # in ppm
-            eta=0.5,
+        isotope = "13C",
+        isotropic_chemical_shift = 100.0,  # in ppm
+        shielding_symmetric = SymmetricTensor(
+            zeta = 70.0,  # in ppm
+            eta = 0.5,
         ),
     )
+    my_sites = [H_site, C_site]
+
+Note that isotopes in ``mrsimulator`` are specified with a string that starts
+with the isotope's mass number followed by its element symbol.
 
-We now have two variables, ``H_site`` and ``C_site``, which are :ref:`site_api` objects. ``H_site``
-represents a proton site with no chemical shift. ``C_site`` represents a carbon-13 site with
-a chemical shift of 100 ppm as well as a shielding component represented by :ref:`sy_api`
-object. We parametrize tensors using the Haeberlen convention.
+In the code above, you created two :ref:`site_api` objects in the variables
+``H_site`` and ``C_site``. The ``H_site`` variable represents a proton site with a
+(default) chemical shift of  zero.  The ``C_site`` variable represents a
+carbon-13 site with a chemical shift of 100 ppm and a shielding
+component represented by a :ref:`sy_api` object. We parametrize tensors using
+the Haeberlen convention. All spin interaction parameters, e.g., isotropic
+chemical shift and other coupling parameters, are initialized to zero by
+default. Additionally, the default Site isotope is ``1H``. In the code above, for
+example, you could have used ``H_site = Site()``.
```

Comment:
Not a huge fan of saying a default proton site can be instantiated with `Site()`. I think mentioning the default site isotope is `1H`, but we should encourage users to be spesific/verbose.

Reply:
you can delete the "for example " part of the sentence